fuck.js

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env node
 
+/// Usage: jsfuck inputfile.js [true|false] > outputfile.js
```

Comment:
Looks interesting! But can you please add some more information to the possible options?